test/apiv2/python/rest_api/test_v2_0_0_pod.py

```diff
@@ -60,6 +60,14 @@ def test_pod_start_conflict(self):
         start = r.json()
         self.assertGreater(len(start["Errs"]), 0, r.text)
 
+        r = requests.post(self.uri(f"/pods/{pod_name[0]}/stop"))
+        self.assertEqual(r.status_code, 200, r.text)
+
+        r = requests.delete(self.uri(f"/pods/{pod_name[0]}"))
+        self.assertEqual(r.status_code, 200, r.text)
+        out = r.json()
+        self.assertEqual(out["Err"], "")
```

Comment:
As coded out is no longer json.

Reply:
would `self.assertIsInstance(r.content, str)` work?